playbooks/packer-functional-public-clouds/run.yaml

```diff
@@ -10,7 +10,9 @@
           set -ex
           set -o pipefail
           pip install python-openstackclient
-          flavor=$(openstack flavor list -f value -c ID -c RAM -c VCPUs --sort-column RAM --sort-column VCPUs | head -n 1 | awk '{print $1}')
+          # Using second smallest flavor, because first one might cause testing failed.
+          # See https://github.com/theopenlab/openlab/issues/80
+          flavor=$(openstack flavor list -f value -c ID -c RAM -c VCPUs --sort-column RAM --sort-column VCPUs | awk 'NR==2{print $1}')
```

Comment:
maybe you can try like below, it is looks a more generic way: 

```
MEDIUM_FLAVOR=$(openstack flavor list -f value -c ID -c RAM -c VCPUs --sort-column RAM --sort-column VCPUs | awk '{if($2>=2048 && $3>=2){print $1}}' | head -n 1)
```
it means the fist flavor with mem >= 2048m and vCPU >=2 :)

Reply:
I think we should't add so many `magic` code, just keep command simple, and awk $2 $3 have to match the first command order of  `-c` option, that's difficult to maintain for other developers.

Reply:
For different clouds, the flavors maybe different, this way select the flavor only according to mem and  vCPUs to avoid confusion. maybe just add a NOTE for this statement to explain for other developers. 
but I am also ok with your change.

Reply:
Why not increase the timeout?